core/src/test/java/nl/nn/adapterframework/http/rest/ApiListenerTest.java

```diff
@@ -56,6 +56,16 @@ public void testProducesUpperCase() throws ConfigurationException {
 		assertEquals("XML", listener.getProduces());
 	}
 
+	@Test
+	public void testProducesPdfWithCharset() throws ConfigurationException {
+		listener.setProduces("PDF");
+		listener.setCharacterEncoding("utf-8");
+		listener.configure();
+
+		assertEquals("PDF", listener.getProduces());
+		assertEquals("application/pdf;charset=UTF-8", listener.getContentType());
+	}
```

Comment:
Dit is een rare test, om een binair formaat te gebruiken en er dan een charset bij te doen.
Als anderen het zien kan het bovendien verwarrend zijn. Je moet zo'n test met een tekst content-type doen.